modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/BlogsPortlet.java

```diff
@@ -84,6 +88,13 @@ protected void doDispatch(
 				_portal.getLiferayPortletRequest(renderRequest),
 				_portal.getLiferayPortletResponse(renderResponse)));
 
+		HttpServletRequest httpServletRequest = _portal.getHttpServletRequest(
+			renderRequest);
+
+		httpServletRequest.setAttribute(
+			CTTimelineKeys.CLASS_NAME, BlogsEntry.class.getName());
+		httpServletRequest.setAttribute(CTTimelineKeys.CLASS_PK, null);
```

Comment:
@ctang3 do we need to set CLASS_PK if it is null

Reply:
Yes - without manually setting it to `null` here, it was being set to `0` elsewhere. This makes the timeline icon not appear. I believe this is also the same issue that Gi mentioned during yesterday's standup too.

Reply:
can you figure out where its being set to 0 and we should fix that instead?

Reply:
Hey guys, I did it like this https://github.com/liferay-site-management/liferay-portal/pull/1501/commits/33dfbb8374f12326493a4eecd5041d0d2c5e02ea, 

I noticed that some entities, like ddmStruture and ddmTemplate were returning empty responses on the api when we passed 0 as the classPK. JournalArticles worked fine with adding the `&classPK=0` at the end of the query, but some other didn't. 
Doing this I just passed the class_PK when it was necessary, like when we want to view/edit a specific entry
cc: @ctang3 @david-truong